drafts/prometheus-alertmanager-rhocp-open-liberty.adoc

```diff
@@ -0,0 +1,207 @@
++
+```
+❯ oc create secret generic alertmanager-alertmanager-main --from-file=alertmanager.yaml
+secret/alertmanager-alertmanager-main created
+```
++
+In this example, the name of our secret is `alertmanager-alertmanager-main` because our Alertmanager name is `alertmanager-main`, so we prefix the secret name with `alertmanager`.
++
+. Check that the service has started successfully.
++
+```
+❯ oc get svc -n prometheus-operator
+NAME                    TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)                      AGE
+alertmanager-operated   ClusterIP   None         <none>        9093/TCP,9094/TCP,9094/UDP   18m
+prometheus-operated     ClusterIP   None         <none>        9090/TCP                     64m
+```
++
+. Once your pods are up and running, expose the route:
++
+```
+❯ oc expose svc/alertmanager-operated -n prometheus-operator
+route.route.openshift.io/alertmanager-operated exposed
+❯ oc get route -n prometheus-operator
+NAME                    HOST/PORT                                                                  PATH   SERVICES                PORT   TERMINATION   WILDCARD
+alertmanager-operated   alertmanager-operated-prometheus-operator.apps.jenniferc.os.fyre.ibm.com          alertmanager-operated   web                  None
+prometheus-operated     prometheus-operated-prometheus-operator.apps.jenniferc.os.fyre.ibm.com            prometheus-operated     web                  None
+```
+
+You can now access the Alertmanager UI. Since we haven't configured Prometheus to send the Alertmanager any alerts yet, you won't see any alert groups at the moment. We'll do that next.
+
+=== Receiving alerts via Prometheus Alertmanager
+Now that the Alertmanager is set up, we need to configure Prometheus to talk to it.
+
+. First, we'll need to expose the Alertmanager port. On the RHOCP web console, click on Networking > Services > Create Service. Create a ClusterIP service:
++
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: alertmanager-example-service
+  namespace: prometheus-operator
+spec:
+  type: ClusterIP
+  ports:
+  - name: web
+    port: 9093
+    protocol: TCP
+    targetPort: web
+  selector:
+    alertmanager: alertmanager-main
+```
++
+The `selector.alertmanager` must match your Alertmanager's name, if you changed it from the default value.
++
+
+. On the RHOCP web console, click on Operators > Installed Operators > Prometheus Operator > Prometheus, then click on the name of your Prometheus instance.
+. In the YAML, add your new service to the `alertmanagers` for Prometheus to talk to.
++
+```
+spec:
+  alerting:
+    alertmanagers:
+      - name: alertmanager-example-service
+        namespace: alertmanager
+        port: web
+```
++
+. Go to your Prometheus route, then click on Alerts. Ensure that at least one alert is firing.
+. Verify that Alertmanager has received the alert by going to the Alertmanager route.
```

Comment:
Not sure on this one. Should "the" be added before "Alertmanager"?:
"Verify that **the** Alertmanager has received the alert by going to the Alertmanager route."